Classes/Controller/FrontEndEditorController.php

```diff
@@ -17,15 +17,10 @@
  */
 class FrontEndEditorController extends ActionController
 {
-    private Context $context;
-
-    private TeaRepository $teaRepository;
-
-    public function __construct(Context $context, TeaRepository $teaRepository)
-    {
-        $this->context = $context;
-        $this->teaRepository = $teaRepository;
-    }
+    public function __construct(
+        private readonly Context $context,
+        private readonly TeaRepository $teaRepository,
+    ) {}
```

Comment:
The php Code Sniffer and php-cs-fixer have different rules.
I'd opt for removing the PHP Code Sniffer in favour of php-cs-fixer and only use one tool for CGL.

Reply:
Let's do that in a separate PR though, please.